src/components/home/Circle.js

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image";
+
+const Circle = ({ width, height, clip, image }) => {
+  let rectClip;
+  if (clip === "left") {
+    rectClip = { clipPath: "polygon(0% 0%, 0% 100%, 50% 100%, 50% 0%)" };
+  } else {
+    rectClip = { clipPath: "polygon(100% 0%, 100% 100%, 50% 100%, 50% 0%)" };
+  }
+
+  return (
+    <div
+      className="flex justify-center items-center bg-winc-pink rounded-full outline outline-4 outline-offset-8 outline-winc-pink w-full h-full relative"
+      style={{ width: width, height: height }}
+    >
+      <Image
+        src={image}
+        width={width * 1.1}
```

Comment:
ok so you can remove the 1*1 coz that is an asset issue. I exported and pushed the new images(homeWinc and homeWinc2) into your branch. When you work with that image it should be big enough.